inc/core/templates/theme-info-page.php

```diff
@@ -217,6 +214,23 @@
 
 					</div><!-- /Colour Palettes -->
 
+
+					<div class="bcgov-block-theme-grid-content card">
```

Comment:
do you think it is worth while to add a manual way of doing this, ie if it doesn't support it, you can add this class `insert-your-class-here`.  or do you think this will cover 99% of the images

Reply:
It's not a bad idea. I have to dig into images as used by other blocks. I think the image and media-text blocks will handle the majority of use cases (possibly ~99%) but there's always going to be the edge to deal with. I think print mode will need ongoing review and possible updating based on usage. Perhaps it can be a repetitive research ticket ie: once per quarter review of sites to see if the print mode is staying relevant...